event_topics/models.py

```diff
@@ -11,3 +11,11 @@ class EventTopic(models.Model):
 
     def __str__(self):
         return self.name
+    
+class EventSubTopic(models.Model):
+    name = models.CharField(max_length=200, null=True)
+    slug = models.CharField(max_length=200, null=True)
+    event_topic_id = models.ForeignKey(EventTopic, on_delete=models.CASCADE, null=True)
+
+    def __str__(self):
+        return self.name
```

Comment:
You `name` field is defined as `null=True`, so `self.name` can be `None`. It is illegal for `__str__` (must always be string).

Reply:
Sorry for the delay. I have added the suggested changes. Pls review.